src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -350,7 +350,7 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
                content_version=None, admin_username="azureuser", agent_count="3",
                agent_vm_size="Standard_D2_v2", location=None, master_count="3",
                orchestrator_type="dcos", service_principal=None, client_secret=None, tags=None,
-               custom_headers=None, raw=False,
+               custom_headers=None, windows=False, admin_password="", raw=False,
```

Comment:
Suggest update the params.py file to make `windows` as a flag, rather with values `true` and `false`